IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcTranslationOperation/Documentation.md

```diff
@@ -0,0 +1 @@
+A translation operation specifies an offset in the coordinate reference system. It does not specify any conversion or distortions only a translation.
```

Comment:
I'd add an additional comment:

With it, only the point of origin of the context is defined; the coordinate axes of the context are not necessarily equal to the underlying CRS and are defined with ...